pl.js

```diff
@@ -0,0 +1,3139 @@
+
+  // Original text: "Type"
+  srType: 'Typ',
+
+  // Original text: "Status"
+  pbdStatus: 'Status',
+
+  // Original text: "Connected"
+  pbdStatusConnected: 'Połączone',
+
+  // Original text: "Disconnected"
+  pbdStatusDisconnected: 'Rozłączone',
+
+  // Original text: "Connect"
+  pbdConnect: 'Połącz',
+
+  // Original text: "Disconnect"
+  pbdDisconnect: 'Rozłącz',
+
+  // Original text: "Forget"
+  pbdForget: 'Zapomnij',
+
+  // Original text: "Shared"
+  srShared: 'Udostępnione',
+
+  // Original text: "Not shared"
+  srNotShared: 'Nieudostępnione',
+
+  // Original text: "No storage detected"
+  pbdNoSr: 'Nie wykryto macierzy dyskowej',
+
+  // Original text: "Name"
+  patchNameLabel: 'Nazwa',
+
+  // Original text: "Install all patches"
+  patchUpdateButton: 'Instaluj wszystkie łatki',
+
+  // Original text: "Description"
+  patchDescription: 'Opis',
+
+  // Original text: "Applied date"
+  patchApplied: "Applied date",
+
+  // Original text: "Size"
+  patchSize: 'Rozmiar',
+
+  // Original text: "Status"
+  patchStatus: 'Status',
+
+  // Original text: "Applied"
+  patchStatusApplied: 'Applied',
+
+  // Original text: "Missing patches"
+  patchStatusNotApplied: 'Brakujące łatki',
+
+  // Original text: "No patch detected"
+  patchNothing: 'Nie wykryto łatek',
+
+  // Original text: "Release date"
+  patchReleaseDate: 'Data wydania',
+
+  // Original text: "Guidance"
+  patchGuidance: 'Guidance',
+
+  // Original text: "Action"
+  patchAction: 'Akcja',
+
+  // Original text: "Applied patches"
+  hostAppliedPatches: 'Łatki zostały zastosowane',
+
+  // Original text: "Missing patches"
+  hostMissingPatches: 'Brakujące łatki',
+
+  // Original text: "Host up-to-date!"
+  hostUpToDate: 'Host posiada najnowsze łatki!',
+
+  // Original text: "Refresh patches"
+  refreshPatches: 'Odśwież łatki',
+
+  // Original text: "Install pool patches"
+  installPoolPatches: 'Instaluj pulę łatek',
+
+  // Original text: "Default SR"
+  defaultSr: 'Domyślny SR',
+
+  // Original text: "Set as default SR"
+  setAsDefaultSr: 'Ustaw domyślny SR',
+
+  // Original text: "General"
+  generalTabName: 'General',
+
+  // Original text: "Stats"
+  statsTabName: 'Statystyki',
+
+  // Original text: "Console"
+  consoleTabName: 'Konsola',
+
+  // Original text: "Container"
+  containersTabName: 'Kontener',
+
+  // Original text: "Snapshots"
+  snapshotsTabName: 'Snapshots',
+
+  // Original text: "Logs"
+  logsTabName: 'Logi',
+
+  // Original text: "Advanced"
+  advancedTabName: 'Zaawansowane',
+
+  // Original text: "Network"
+  networkTabName: 'Sieć',
+
+  // Original text: "Disk{disks, plural, one {} other {s}}"
+  disksTabName: 'Dysk{disks, plural, one {} other {s}}',
+
+  // Original text: "halted"
+  powerStateHalted: 'Zatrzymany',
+
+  // Original text: "running"
+  powerStateRunning: 'Działający',
+
+  // Original text: "suspended"
+  powerStateSuspended: 'Uśpiony',
+
+  // Original text: "No Xen tools detected"
+  vmStatus: 'Nie wykryto Xen tools'ów',
```

Comment:
You can use double quotes when you need simple quotes in translation. Eg this will become:
```
"Nie wykryto Xen tools'ów",
```